src/utils/inferencedata.jl

```diff
@@ -1,17 +1,83 @@
-using InferenceObjects
-using PosteriorDB
+function inferencedata(m::SampleModel; kwargs...)
 
-import Base: convert
+    # Used keyword arguments
+    used_keywords = [:include_log_likelihood, :include_sample_stats,
+        :include_posterior_predictive]
 
-"""
+    # Default is only posterior values
+    include_log_likelihood = false
+    include_sample_stats = false
```

Comment:
I think it makes more sense to include these by default. In fact, I can't think if a good reason to even offer to not include them. It's recommended that users check the sample stats. They should at the very least check for divergent transitions every time, so we can incentivize that by always including it.

If log-likelihood is present in the draws, it's because they hand-coded it in the model, so it's safe to assume they want those values. If not, they can always discard it from their `InferenceData` after the fact.